terraform/backends/ab2d-dev.s3.tfbackend

```diff
@@ -0,0 +1,6 @@
+bucket         = "ab2d-dev-tfstate"
+key            = "ab2d-dev/terraform/api/terraform.tfstate"
```

Comment:
Key is defined in provider.tf so does not need to be included here. That will allow us to define other services that use these same tfbackend files with different tfstate keys.